docs/source/quickstart.rst

```diff
@@ -1,81 +1,187 @@
 
-Quickstart
-============
+⚡️ Eureka! Quickstart ⚡️
+================
 
-1. Installation and requirements
------------------------------------
+Want to get up and running with ``Eureka!``, but not really sure where to begin? Keep reading! 
+
+1. Installation 📦
+------------------
 
-Follow the instructions on the :ref:`installation` page.
+The first thing you need to do is install the package, so if you haven't already, take a break from this page and follow the :ref:`installation` instructions (if you have issues be sure to visit the `FAQ page <https://eurekadocs.readthedocs.io/en/latest/installation.html#issues-installing-or-importing-jwst>`_ first). 
```

Comment:
Can you update the FAQ link to a reference like for the installation intructions